.github/actions/unit-tests/prepare.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
@jibi my commit is a little bit different than the changes being backported :confused:

Reply:
this file should have been removed altogether, fixed now